projects/plugins/protect/src/js/components/modal/styles.module.scss

```diff
@@ -7,25 +7,25 @@
     display: flex;
     justify-content: center;
     align-items: start;
-    padding-top: 98px;
+    padding-top: calc( var( --spacing-base ) * 12 + 2 ); // 98px
```

Comment:
This calculation doesn't appear to be working so the `padding-top` isn't being applied:

<img width="1328" alt="Screen Shot 2022-11-02 at 12 30 42" src="https://user-images.githubusercontent.com/43220201/199586562-f15d4fb0-8a03-4265-b0b6-d38ed28d11d9.png">

Reply:
Turns out I needed to specify the `px` unit for the `+ 2` here! Resolved via [5f31b9b](https://github.com/Automattic/jetpack/pull/27175/commits/5f31b9b1c3cbd454730415f7adc4004a42e87adf) :+1: